server/blob/blob-objectstorage/src/test/java/org/apache/james/blob/objectstorage/ObjectStorageBlobsDAOTest.java

```diff
@@ -149,5 +150,21 @@ void encryptionWithCustomPayloadCodeCannotBeReadFromUnencryptedDAO() throws Exce
         InputStream clearTextIs = encryptedDao.read(blobId);
         assertThat(clearTextIs).hasSameContentAs(new ByteArrayInputStream(bytes));
     }
+
+    @Test
+    void clearContainerShouldDeleteAllContainerContents() {
+        byte[] bytesOfFirstBlob = "First Blob".getBytes(StandardCharsets.UTF_8);
+        byte[] bytesOfSecondBlob = "Second Blob".getBytes(StandardCharsets.UTF_8);
+        BlobId firstBlobId = testee.save(bytesOfFirstBlob).join();
+        BlobId secondBlobId = testee.save(bytesOfSecondBlob).join();
+
+        testee.clearContainer();
+        SoftAssertions.assertSoftly(softly -> {
+            softly.assertThat(testee.read(firstBlobId))
+                .isEqualTo(new ByteArrayInputStream(new byte[]{}));
```

Comment:
you should use `hasContentEqualTo` and please introduce a EMPTY_STREAM constant.